starcounter-include.html

```diff
@@ -203,24 +203,29 @@
          * @param  {String} path Polymer notification path
          * @param  {Mixed} value New value
          */
-        LauncherPartialPrototype._notifyPath = function(path, value) {
+        StarcounterIncludePrototype._notifyPath = function(path, value) {
```

Comment:
This method would be simpler if after line 206 we would do: 

```js
path = path.replace(/^(partial\.)/, "model.").replace(/^(viewModel\.)/, "model.");
```

Then all the if's below can just check `model.Html`, etc.